doc/0.120/_sources/feeds.rst.txt

```diff
@@ -0,0 +1,169 @@
+                <admarkt:image url="http://supabike.com/files/colored/fiets.jpg"/>
+            </admarkt:media>
+            <admarkt:budget>
+                <admarkt:totalBudget>200000</admarkt:totalBudget>
+                <admarkt:dailyBudget></admarkt:dailyBudget>
+                <admarkt:cpc>2</admarkt:cpc>
+            </admarkt:budget>
+            <admarkt:emailAdvertiser>true</admarkt:emailAdvertiser>
+            <admarkt:microTip>Lease 499 per mnd</admarkt:microTip>
+        </admarkt:ad>
+    </admarkt:ads>
+
+
+Scenario's & Questions
+----------------------
+
+Below are some common asked-for scenarios and questions with their explanations/answers.
+
+- Feed file cannot be fetched
+
+When a feed file cannot be fetched, nothing will change on the user's ads.
+It's as if the import didn't happen. Since the file represents the desired
+list of ads to be live, we won't do anything if we can't get the file -
+we cannot read a change in the desired situation.
+
+- Feed file is empty
+
+When a feed file is empty, all ads of the user will be paused. An empty
+file means the desired list of ads to be live is empty, so all active ads
+are paused. Note that this also means that all ads which are in statuses
+BUDGET_REACHED or DAILY_BUDGET_REACHED are also paused.
+
+- Feed file contains only new ads
+
+In the spirit of the feed file being the desired set of ads to be live for
+a user, all currently active ads (including ads in BUDGET_REACHED or DAILY_BUDGET_REACHED)
+will be paused and the supplied ads will be created (with status ACTIVE).
+
+- Editing ads through frontend / API
+
+In the spirit of the feed file being the desired set of ads to be live for
+a user, all ads will be (re)set to their representing feed values. This means
+you can see changes made throuh API or frontend undone after a successful feed import.
+
+- XML does not validate against XSD
+
+If the fetched XML filed does not validate against the XSD there will not be any changes
+to your ads. Existing ads will remain unchanged and no new ads will be created.
+
+- How to validate XML against XSD
+
+Next to various online capabilities where you can provide both your XML and XSD files,
+a way to check quickly and locally is to use a tool called xmllint. With this tool
+you can use our XSD downloaded from :ref:`get_feed_xsd` to test whether your feed is
+working before you let iCAS fetch it using the following command:
+
+.. code-block:: bash
+
+  xmllint --debug --noout --schema /path/to/admarkt1.0.xsd /path/to/yourfeed.xml
+
+For small chunks of XML you can use online validators as well, such as `<http://www.utilities-online.info/xsdvalidation/>`_ or `<https://www.freeformatter.com/xml-validator-xsd.html>`_. Note that these have a
+limit on the size of the XML you can check, but it should be more than enough to be able
+to test correctness of your structure.
+
+- I'm using the same image URLs on an ad, but the images hosted on these URLs have changed.
```

Comment:
While it does describe the problem clearly, the style of the title is out of sync with the rest. 
To be in sync with the rest it should state a problem: 
`Image updates are not in ICAS after successful feed run` 

Then you can go and explain why.